src/AM.Condo/Targets/Test/DockerCompose.targets

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xlmns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<Target Name="DockerComposeTest" Condition=" $(DockerRequired) AND $(DOCKER_COMPOSE_PATHS) != '' ">
```

Comment:
If we want to use DOCKER_COMPOSE_PATHS as an override then it should be created as a prepare step. 

We also need the discovery of dockercompose.test files

Reply:
override is not needed anymore, we are using discovery pattern with DockerRoot path